package.json

```diff
@@ -6,16 +6,16 @@
   "license": "MIT",
   "dependencies": {
     "@babel/eslint-parser": "^7.17.0",
-    "@typescript-eslint/eslint-plugin": "^5.10.2",
-    "@typescript-eslint/parser": "^5.10.2",
+    "@typescript-eslint/eslint-plugin": "^6.1.0",
+    "@typescript-eslint/parser": "^6.1.0",
     "eslint-import-resolver-webpack": "^0.13.2",
     "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-jest": "^26.0.0",
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.28.0",
     "prettier": "2.5.1",
-    "typescript": "^4.5.5"
```

Comment:
I think there's an honest question of whether we should even have `typescript` as a dependency of this repo. I'd make the argument that it should be removed since teams dictate what version of TypeScript their codebase is using, not the linter we choose to use.

Reply:
Actually just gunna merge this for now and we can figure out the answers to those questions later